### ISSUE TITLE
docs: site contains aria and a11y issues

### ISSUE THREAD

**Issue by Splaktar** [angular/material#11452]
## Bug, enhancement request, or proposal:
Bug

# CodePen and steps to reproduce the issue:
#### [CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd) which demonstrates the issue: 
https://material.angularjs.org/latest/

#### Detailed Reproduction Steps: 
1. Run a Lighthouse A11y Audit
1. Review results

### What is the expected behavior?
The Lighthouse a11y audit should be very close to `100`. 

### What is the current behavior?
We're currently at `86` due to some issues with contrast, repeated non-unique element `id`s, and some other issues that are a problem with the docs site and not our actual components.

### What is the use-case or motivation for changing an existing behavior?
We build accessible components, our docs need to be accessible as well.

### Which versions of AngularJS, Material, OS, and browsers are affected?

- AngularJS: All
- AngularJS Material: 1.1.10
- OS: All
- Browsers: Chrome
